### PR TITLE
🐛 Custom gitmojis url was not set

### PR DESCRIPTION
**Pull request by Ras96**
## Description

Thanks to [🔧 Add gitmojis url config by YunYouJun · Pull Request #640](https://github.com/carloscuesta/gitmoji-cli/pull/640), you can custom `GITMOJIS_URL`.
But It didn't seem to be working.

## Tests

- [x] All tests passed.

### CODE REVIEW

File: src/commands/config/index.js [hunk 1]
```diff
@@ -16,6 +16,9 @@ const config = () => {
     configurationVault.setScopePrompt(
       answers[CONFIGURATION_PROMPT_NAMES.SCOPE_PROMPT]
     )
+    configurationVault.setGitmojisUrl(
+      answers[CONFIGURATION_PROMPT_NAMES.GITMOJIS_URL]
+    )
```
Comment (carloscuesta) [inline src/commands/config/index.js:21]: Great catch! Can you update the test spec to include a case for covering this?

https://github.com/carloscuesta/gitmoji-cli/blob/master/test/commands/config.spec.js

**Review by carloscuesta — COMMENTED** [review]
Great one! 🙏🏼

File: src/commands/config/index.js (hunk 1 above)
Comment (Ras96) [reply src/commands/config/index.js:21]: Thanks for the comment! :+1:

**Review by carloscuesta — APPROVED** [review]
Thanks! ❤️

**Pull request merged**